client/src/components/cards/CardsIndex.jsx

```diff
@@ -44,4 +74,4 @@ function mapDispatchToProps(dispatch) {
   }
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(CardsIndex);
+export default connect(mapStateToProps, mapDispatchToProps)(withStyles(styles)(CardsIndex));
```

Comment:
is this necessary? ```withStyles(styles)```

Reply:
withStyles is a higher order component used to inject CSS into the DOM. It appears to be the recommended method to style components in material design.